scripts/deployment/README.md

```diff
@@ -0,0 +1,13 @@
+# Deployment script
```

Comment:
Consider adding a brief section about installation / dependencies / run, and adding a text file with dependencies, usually called `requirements.txt` with content like this:

```
psycopg2==2.9.6
```